tests/test_result_visitor.py

```diff
@@ -0,0 +1,23 @@
+import unittest
```

Comment:
Please, do not use unittest library. It has limited support from the Pytest side and may be deprecated in the future.